packages/app/src/components/nominationBanner/style.css

```diff
@@ -17,7 +17,20 @@
 }
 
 .decline-button {
+  background-color: #f72314;
+  color: #ffffff;
+  border: #8a8a8a;
+  font-size: 0.82rem;
+  font-weight: bold;
+  text-justify: center;
+  padding: 0px 10px 10px 10px;
+  height: 35px;
   border-radius: 4rem;
+  cursor: pointer;
+}
+
+.decline-button:hover {
+  background-color: #d4190c;
 }
```

Comment:
Generally, I'd expect these styles to be shared for every button. The CSS framework we're using (Milligram) provides its own styles for the `.button` class, but if we have any standard overrides we want to apply across multiple components, I think those should be added to [App.css](https://github.com/the-difference-engine/ksf/blob/develop/packages/app/src/App.css).

Reply:
Okay, Ill update App.css to includes styling for .button.